server/controllers/MissionController.js

```diff
@@ -0,0 +1,20 @@
+const { getMission, updateMission } = require('../store/missions');
+const { createMissionUpdate } = require('../store/mission_updates');
+const { getVehicle, updateVehicleStatus } = require('../store/vehicles');
+
+
+const command = async (req, res) => {
+  const { user_id, mission_id, command} = req.query;
+  const mission = await getMission(mission_id);
+  const vehicle = await getVehicle(mission.vehicle_id);
+
+  if (command === 'takeoff_pickup' && vehicle.status === 'waiting_pickup'){
```

Comment:
Add a test that user_id matches the mission's user_id

Reply:
I did this and somehow didn't push it. Just did.